docs/guide/token_factory.md

```diff
@@ -0,0 +1,847 @@
+
+After scaffolding the denom map, it is a good time to make a first git commit:
+
+```bash
+git add .
+git commit -m "Add tokenfactory module and denom map"
+```
+
+This git commit saves all staged changes. This commit captures a snapshot of the project that you can come back to in case something goes wrong with the following steps.
+
+## Remove Delete Messages Functionality
+
+Since a created denom is subsequently handled by the `bank` module like any other native denom, the denom must not be deletable. To prevent the denom from being deleted, remove all references to the delete action of the scaffolded CRUD type.
+
+In order to remove the functionality to delete token, you must remove these delete functions from proto, client, keeper, and handler files as shown in the next sections.
+
+Take care to retain comments that were created by Starport. For example:
+
+```
+// this line is used by starport scaffolding #1
+```
+
+### Proto
+
+In the proto file `proto/tokenfactory/tx.proto`, remove this code:
+
+```proto
+rpc DeleteDenom(MsgDeleteDenom) returns (MsgDeleteDenomResponse);
+```
+
+Then, remove the `MsgDeleteDenom` and `MsgDeleteDenomResponse` messages.
+
+### Client
+
+Navigate to the client in `x/tokenfactory/client` and make these changes:
+
+- In the `x/tokenfactory/client/cli/tx_denom_test.go` file, remove the entire `TestDeleteDenom()` function.
+- In the `x/tokenfactory/client/cli/tx_denom.go` file, remove the entire `CmdDeleteDenom()` function.
+- In the `x/tokenfactory/client/cli/tx.go` file, remove the line that adds the delete command:
+
+    ```go
+    cmd.AddCommand(CmdDeleteDenom())
+    ```
+
+### Keeper
+
+In the `x/tokenfactory/keeper` directory, update the files that contain the delete denom functionality:
+
+- In the `x/tokenfactory/keeper/denom_test.go` file, remove the `TestDenomRemove()` function.
+- In the `x/tokenfactory/keeper/denom.go` file, remove the entire `RemoveDenom()` function.
+- In the `x/tokenfactory/keeper/msg_server_denom_test.go` file, remove the `TestDenomMsgServerDelete()` function.
+- In the `x/tokenfactory/keeper/msg_server_denom.go` file, remove the `DeleteDenom()` function.
+
+### Types
+
+The `types` directory defines functions and validations that describe the format of the blockchain data. You must remove the delete denom functionality from the codec, the message denom test, and the message denom file.
+
+- In the `x/tokenfactory/types/codec.go` file, remove the codec and interface registrations for `MsgDeleteDenom`.
+- In the `x/tokenfactory/types/messages_denom_test.go` test, remove the `TestMsgDeleteDenom_ValidateBasic()` function.
+- In the `x/tokenfactory/types/messages_denom.go` file, remove the entire part that references `MsgDeleteDenom()`.
```

Comment:
I found more than one reference, shall we add instructions to remove construct on line 11, too?  or adjust language for "entire part" and say "all references to"?

```
const (
    TypeMsgCreateDenom = "create_denom"
    TypeMsgUpdateDenom = "update_denom"
    TypeMsgDeleteDenom = "delete_denom"
)
```